css/cssom-view/scrollIntoView-direction-rtl.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<title>CSSOM View - scrollIntoView considers direction:rtl</title>
+<meta charset="utf-8">
+<link rel="author" title="Cathie Chen" href="mailto:cathiechen@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/cssom-view/#dom-element-scrollintoview">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<style>
+.box {
+  float: left;
+  width: 200px;
+  height: 200px;
+}
+#scroller {
+  direction: rtl;
+  overflow-x: scroll;
+  width: 300px;
+  height: 215px;
+}
+#container{
+  width: 600px;
+  height: 200px;
+}
+#target {
+  background-color: #ff0;
+}
+</style>
+<body>
+<div id="scroller">
+  <div id="container">
+    <div class="row">
+      <div class="box"></div>
+      <div class="box" id="target"></div>
+      <div class="box"></div>
+    </div>
+  </div>
+</div>
+
+<script>
+var target = document.getElementById("target");
```

Comment:
Can you add a comment to explain that in horizontal-lr and direction: rtl, the overflow direction is leftward / downward and that inline == x. Also x decreases leftward, so is nonpositive etc

Reply:
Done